security/security_policy_compliance/policy-checks.sh

```diff
@@ -39,65 +39,80 @@ function get_volume_attribute {
   echo $(echo "$HAYSTACK" | grep -e "$NEEDLE:" | cut -d : -f2 | sed 's/^ *//g')
 }
 
+function validate_time_machine {
+  start "Checking Time Machine Backup Disk Encryption"
+
+  # Replace spaces with three underscores so we can use spaces as a delimiter in an array
+  TIME_MACHINE_CONFIGURATION=$(tmutil destinationinfo)
+  TIME_MACHINE_VOLUMES=$(tmutil destinationinfo | grep -e "Name" | sed 's/Name.*: //' | sed -e "s/ /___/g")
+
+  if [ "${TIME_MACHINE_CONFIGURATION}" == "tmutil: No destinations configured." ]; then
+    success "Backups are not configured. Encryption not required."
+  elif [ "${TIME_MACHINE_VOLUMES}" == "" ]; then
+    warning "No backup disks configured with Time Machine." "https://github.com/sparkbox/standard/blob/main/security/security_policy_compliance/timemachine.md#first-time-setup"
+  fi
+
+  for VOLUME in $TIME_MACHINE_VOLUMES; do
+    VOLUME_NAME=$(echo ${VOLUME} | sed -e "s/___/ /g")
+    VOLUME_INFO=$(diskutil info "${VOLUME_NAME}" 2> /dev/null)
+    VOLUME_TYPE=$(get_volume_attribute "Type (Bundle)" "${VOLUME_INFO}")
+
+    case $VOLUME_TYPE in
+      # APFS volumes will identify their encryption status under "FileValt"
```

Comment:
Yeah I'm going to file this under a "my bad".

```suggestion
      # APFS volumes will identify their encryption status under "FileVault"
```